components/esb-tools/plugins/org.wso2.integrationstudio.gmf.esb.diagram/src/org/wso2/integrationstudio/gmf/esb/diagram/part/EsbMultiPageEditor.java

```diff
@@ -1308,10 +1344,10 @@ public void run() {
                         SourceError sourceError;
                         
                         if (currArtifactType.toString().equals(MOCK_SERVICE) || 
-                        		currArtifactType.toString().equals(SYNAPSE_UNIT_TEST)) {
```

Comment:
Shall we keep the existing formatting? otherwise, it's hard to see the actual changes. If needed we can format using separate commit

Reply:
fixed